include/video.h

```diff
@@ -78,12 +78,12 @@ void GFX_SwitchFullScreen(void);
 bool GFX_StartUpdate(uint8_t * &pixels, int &pitch);
 void GFX_EndUpdate( const uint16_t *changedLines );
 void GFX_GetSize(int &width, int &height, bool &fullscreen);
-void GFX_UpdateMouseState();
 void GFX_LosingFocus();
 void GFX_RegenerateWindow(Section *sec);
+
+void GFX_SetMouseCapture(const bool capture);
+void GFX_SetMouseCursor(const bool visible);
```

Comment:
`GFX_SetMouseCapture` -> the function name is self-document; I understand I'm capturing or uncapturing:

GFX_SetMouseCapture(true) -> capture
GFX_SetMouseCapture(false) -> uncapture

But `GFX_SetMouseCursor` is less self documenting:

GFX_SetMouseCursor(true)
GFX_SetMouseCursor(false)

What am I changing about the cursor?

Suggest the name use an adjective:

GFX_SetMouseVisibility(true)
GFX_SetMouseVisibility(false)

Or be reworded as an action / verb:

GFX_ShowMouseCursor(true)
GFX_ShowMouseCursor(false)

Reply:
`GFX_SetMouseVisibility` looks best to me. It is in similar style as `GFX_SetMouseCapture` and others, it also plays well with variable names in `mouse.cpp`. Changed.

Reply:
> It is in similar style 

Ahh yes.. `SetMouse<Adjective>` :+1: